api/src/opentrons/api/session.py

```
@@ -237,6 +246,7 @@ def on_command(message):
         return res
 
     def refresh(self):
+        assert not self._loop.is_closed()
```

Is this recoverable? If so, I'd rather recover than assert. At best, the assert will raise a weird, non-actionable error in the app; at worst it'll fail silently